.perltidyrc

```diff
@@ -0,0 +1,27 @@
+-et=4
+-dt=4
+-tabs
```

Comment:
```suggestion
```
The documentation states that `-et` is preferred over `--tabs`, so I think it's safe to remove this. (I don't know if it ever even did anything because of the missing `-` :))